cpp/oneapi/dal/data/undirected_adjacency_array_graph.cpp

```diff
@@ -31,23 +31,11 @@ undirected_adjacency_array_graph<VertexValue, EdgeValue, GraphValue, IndexType,
                                                                   IndexType,
                                                                   Allocator>) {}
 
-template class undirected_adjacency_array_graph<empty_value,
-                                                empty_value,
-                                                empty_value,
-                                                std::int64_t,
-                                                std::allocator<empty_value>>;
-
-template class undirected_adjacency_array_graph<empty_value,
-                                                empty_value,
-                                                empty_value,
-                                                std::int32_t,
-                                                std::allocator<empty_value>>;
-
-using graph64 = undirected_adjacency_array_graph<empty_value,
-                                                 empty_value,
-                                                 empty_value,
-                                                 std::int64_t,
-                                                 std::allocator<empty_value>>;
+template class ONEAPI_DAL_EXPORT undirected_adjacency_array_graph<empty_value,
+                                                                  empty_value,
+                                                                  empty_value,
+                                                                  std::int32_t,
+                                                                  std::allocator<empty_value>>;
 
 using graph32 = undirected_adjacency_array_graph<empty_value,
```

Comment:
'32' may not be clean enough as the name
Also, does it mean that we do not support 64 bit for now?

Reply:
@andreynv 
this using purpose used only in cpp file for compact instance of graph service functions.

Reply:
it does not matter, if it is implementation which will be open sourced or api. The expectation is that both apis and implementations will follow consistent and clean style of naming which will help external and internal contributors quickly learn the content and bring extensions in the same style

Reply:
There are no conventions for such case. This using declared in cpp. Hence, It will be not used somewhere else. 
I can rename it for index32_undirected_adjacency_array_graph or for default_undirected_adjacency_array_graph